### PR TITLE
Downstream mattermost-plugin-starter-template

### PR DESCRIPTION
**Pull request by kaakaa**
Downstream changes from https://github.com/mattermost/mattermost-plugin-starter-template 

- Add `plan.yml` for sync and `make sync` with starter-template
- Disable `fieldalignment` of govet
- Change setup script of webapp test from `js` to `tsx`

### CODE REVIEW

File: build/manifest/main.go [hunk 1]
```diff
@@ -114,7 +114,7 @@ func applyManifest(manifest *model.Manifest) error {
 
 		// write generated code to file by using Go file template.
 		if err := ioutil.WriteFile(
-			"server/plugin/manifest.go",
+			"server/manifest.go",
```
Comment (hanzei) [inline build/manifest/main.go:117]: Is that change intended? `server/manifest.go` doesn't seem to be checked in.

File: webapp/tests/setup.tsx [hunk 2]
```diff
@@ -1,11 +1,14 @@
-import '@babel/polyfill';
-import Adapter from 'enzyme-adapter-react-16';
-import {configure} from 'enzyme';
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
```
Comment (hanzei) [inline webapp/tests/setup.tsx:2]: Let's drop these lines

**Review by hanzei — CHANGES_REQUESTED** [review]
Thanks for keeping the plugin up to date :+1:

File: build/manifest/main.go (hunk 1 above)
Comment (kaakaa) [reply build/manifest/main.go:117]: Good catching 👍  Intended, but this doesn't seem to make sense. I revert it.

**Review by hanzei — APPROVED** [review]
